manifests/kube-state-metrics-prometheusRule.yaml

```diff
@@ -40,3 +40,26 @@ spec:
       for: 15m
       labels:
         severity: critical
+    - alert: KubeStateMetricsShardingMismatch
```

Comment:
The metrics for this are only available in master right now, so this won't do anything. Are we sure we are pinning to latest release?

Reply:
We were pinning to `release-1.9` for ksm deployment but to `master` for mixin. Now it should be fixed. Great catch!

Reply:
Yeah we didn't used to have any diff between master and other branches, but we recently merged these cool metrics and alerts. :)